### PR TITLE
Change sairedis commit ID to latest

### PR DESCRIPTION
**Pull request by yuriilisovskyi**

### CODE REVIEW

File: sai.env [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-SAIREDIS_ID=606703e
+SWSS_COMMON_ID=be425ed
+SAIREDIS_ID=eb24302
 
 # SAI version:
 #   Branch v1.12
 #   May 17, 2023
 
-SAI_ID=92875f9
+SAI_ID=ad12e9e
```
Comment (andriy-kokhan) [inline sai.env:13]: why this has been changed?

Comment (yuriilisovskyi) [reply sai.env:13]: SAI latest commit on v1.12 branch is https://github.com/opencomputeproject/SAI/commit/ad12e9eac34996501691d7a159391e25d4353c46

Comment (andriy-kokhan) [reply sai.env:13]: Should be the same commit as used in SONiC master

File: npu/broadcom/BCM56850/saivs/Dockerfile [hunk 2]
```diff
@@ -28,7 +28,7 @@ RUN git clone https://github.com/sonic-net/sonic-sairedis.git \
         && sed -i "s/^SUBDIRS = py2 py3/SUBDIRS = py3/" pyext/Makefile.am \
         && ./autogen.sh \
         && export DEB_BUILD_PROFILES=nopython2 \
-        && dpkg-buildpackage -us -uc -b --target=binary-syncd-vs --jobs=auto \
+        && dpkg-buildpackage -us -uc -b -Psyncd,vs,nopython2 --jobs=auto \
```
Comment (andriy-kokhan) [inline npu/broadcom/BCM56850/saivs/Dockerfile:31]: Do we still need L30 ? The same in all other dockerfiles

Comment (andriy-kokhan) [inline npu/broadcom/BCM56850/saivs/Dockerfile:31]: Even PHY targets are not the part of CI at the moment, please make the same changes for phy/broadcom/BCM81724/

**Review by andriy-kokhan — APPROVED** [review]

**Pull request merged**